pkg/app/app.go

```diff
@@ -25,10 +26,16 @@ var (
 	// routed.
 	NotFound UI = &notFound{}
 
-	routes = make(map[string]UI)
-	uiChan = make(chan func(), 256)
+	routes   = make(map[string]UI)
+	routesRe = make([]regexRoute, 0)
+	uiChan   = make(chan func(), 256)
 )
 
+type regexRoute struct {
```

Comment:
I would like to keep the Golang naming for regular expressions.
```go
type regexpRoute struct
```

Reply:
good suggestion. will do

Reply:
good suggestion. will do